integreat_compass/cms/templates/index.html

```diff
@@ -0,0 +1,16 @@
+{% extends "_base.html" %}
+{% load i18n %}
+{% block title %}
+    {% translate "Home" %}
+{% endblock title %}
+{% block content %}
+    <div class="absolute top-0 right-0 bottom-0 left-0 flex justify-center items-center bg-background-900 z-1">
+        <div class="flex flex-col justify-center gap-4 text-center">
+            <h1 class="font-bold text-5xl drop-shadow-md mb-12 text-white">{% translate "Arriving, reimagined." %}</h1>
+            <a href="#"
+               class="bg-redMuted hover:bg-red-500 text-white font-bold text-3xl py-4 px-4 rounded-2xl">{% translate "Discover Integreat Compass" %}</a>
+            <a href="#"
+               class="bg-white hover:bg-gray-200 font-bold text-3xl py-4 px-4 rounded-2xl">{% translate "Add new offer" %}</a>
+        </div>
+    </div>
```

Comment:
Maybe this is out of cope for this PR, but I think we could already think about which icon-library we want to use and could already the chevron-down that is displayed in the design. If this is out of scope, I would suggest we open a new issue for this

Reply:
Good idea.

Though I actually have some questions about the start page - is it even necessary? What is the advantage of using it instead of just showing the dashboard directly?

I also do not get the chevron down, is the start page supposed to be reachable by scrolling up from the dashboard?

No matter how I think about it, this just feels like an additional click and like it is not offering anything of value.

Reply:
Yes, I agree. That was the second point on my list to ask later. It also doesn't make a lot of sense to me, but I was giving the designer the benefit of the doubt and thought that maybe we're not seeing something :)

Reply:
> we could already think about which icon-library we want to use

TBH I would just go with lucide again, those have been working pretty flawlessly afaik?

Reply:
Yes, I don't know of any issues either and would also just go with lucide again

Reply:
@JoeyStk I know it's not required anymore, but I've left the index page in as a placeholder until the Dashboard and/or all offers views are created.

Icons are now working as well.